Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelemetryDiagnosticControls.java

```diff
@@ -1,44 +1,45 @@
 package tddmicroexercises.telemetrysystem;
 
-public class TelemetryDiagnosticControls
-{
+import tddmicroexercises.*;
+
+public class TelemetryDiagnosticControls {
     private final String DiagnosticChannelConnectionString = "*111#";
-    
-    private final TelemetryClient telemetryClient;
-    private String diagnosticInfo = "";
+    private String diagnosticInfo;
 
-        public TelemetryDiagnosticControls()
-        {
-            telemetryClient = new TelemetryClient();
-        }
-        
-        public String getDiagnosticInfo(){
-            return diagnosticInfo;
+    private final Client telemetryClient;
+    private final TelemetryTransmissionControl telemetryTransmissionControl;
+
+    public TelemetryDiagnosticControls() {
+        telemetryClient = new TelemetryClient();
+        diagnosticInfo = "";
+
+        telemetryTransmissionControl = new TelemetryTransmissionControlImpl();
+    }

```

Comment:
This violates Dependency Injection Principle
pass instances as constructor params

Reply:
Oh yeah missed it. Addressed https://github.com/abhishek-yelisetti/Racing-Car-Katas/pull/1/commits/d4c7dcfe8d602dce01b64dc411fa76a96865e705